Vic2ToHoI4/Data_Files/blankMod/output/interface/countrypoliticsview.gfx

```diff
@@ -103,11 +103,6 @@ spriteTypes = {
 		name = "GFX_autonomy_annex_icon"
 		textureFile = "gfx/interface/autonomy/autonomy_annex_icon.dds"
 	}
-	
```

Comment:
Don't remove this, it's used for the sphereling functionality

Reply:
Done